setup.py

```diff
@@ -9,7 +9,8 @@
     packages=find_packages(),
     install_requires=[
         'pypif>=2.1.0,<3',
-        'xlrd'
+        'xlrd',
+        'pytest'
```

Comment:
This should go in `tests_require` instead of `install_requires`, but it is usually omitted when tests aren't run through `python ./setup.py test` (which may be deprecated).

Reply:
I'll just omit pytest from here